stable/prometheus-operator/templates/prometheus/rules-1.14/alertmanager.rules.yaml

```diff
@@ -5,14 +5,13 @@ https://github.com/helm/charts/tree/master/stable/prometheus-operator/hack
 */ -}}
 {{- $kubeTargetVersion := default .Capabilities.KubeVersion.GitVersion .Values.kubeTargetVersionOverride }}
 {{- if and (semverCompare ">=1.14.0-0" $kubeTargetVersion) (semverCompare "<9.9.9-9" $kubeTargetVersion) .Values.defaultRules.create .Values.defaultRules.rules.alertmanager }}
-{{- $operatorJob := printf "%s-%s" (include "prometheus-operator.fullname" .) "operator" }}
 {{- $alertmanagerJob := printf "%s-%s" (include "prometheus-operator.fullname" .) "alertmanager" }}
-{{- $namespace := printf "%s" (include "prometheus-operator.namespace" .) }}
+{{- $namespace := .Release.Namespace }}
```

Comment:
that is false change. and ALL similar Namespace changes should be reverted.
The reason is that guy who has included that change - did it manually. he has not added it to PYTHON sync scripts... so sync overwrites them. unfortunately for now it has to be fixed manually.
or you can include PROPER fix by yourself:
instead of that https://github.com/helm/charts/pull/22974
original issue https://github.com/helm/charts/issues/22892

Reply:
Ah, I see. I've adjusted the hack and no namespaces should be changed in this PR. Only the actual dashboards & alerting rules.